src/coreclr/src/inc/palclr_win.h

```diff
@@ -135,10 +135,10 @@
 #define WIN_PAL_ENDTRY_NAKED_DBG
 #endif // defined(ENABLE_CONTRACTS_IMPL) && !defined(JIT64_BUILD)
```

Comment:
JIT64_BUILD is never defined in CoreCLR. I know that it is not related to your change, but it would be nice if you can delete it while you are on it.

Reply:
I was seeing FEATURE_REDHAWK,  Any objection to removing it?

Reply:
It is used for CoreRT. I would like to keep for the time being if possible.